docs/source/ruletypes.rst

```diff
@@ -571,6 +571,13 @@ ElastAlert will use ``fields`` to retrieve stored fields. Both of these are repr
 See https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-fields.html for more details. The fields used come from ``include``,
 see above for more details. (Optional, boolean, default True)
 
+scan_entire_timeframe
```

Comment:
Can you add this to https://github.com/Yelp/elastalert/blob/master/docs/source/ruletypes.rst in the appropriate section (e.g. applicable to all rules or just specific ones)